@xen-orchestra/build-template/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@xen-orchestra/build-template",
```

Comment:
I think we could name this package simply `@xen-orchestra/template` or even `@xen-orchestra/tpl`.

@badrAZ @pdonias What's your opinions on this?

Reply:
IMHO, `template` is better